src/components/app-bar/app-bar.atoms.tsx

```diff
@@ -0,0 +1,19 @@
+import type { HTMLAttributes } from 'react'
+import {
+  ElNavSearchButton,
+  ElNavSearchButtonIcon,
+  ElNavSearchButtonPlaceholder,
+  ElNavSearchButtonShortcutText,
+} from './styles'
+
+export interface NavSearchButton extends HTMLAttributes<HTMLButtonElement> {}
+
+export const NavSearchButton: React.FC<NavSearchButton> = (props) => {
+  return (
+    <ElNavSearchButton {...props}>
+      <ElNavSearchButtonIcon icon="search" />
```

Comment:
**suggestion:** I think we should be passing `role="presentation"` to the icon.

Reply:
yeah, or I think `aria-hidden="true"` is more appropriate since it already has placeholder

Reply:
Ah, of course, yes I'd forgotten about aria-hidden 🤦‍♂️ 👍